tasks/views.py

```diff
@@ -16,31 +16,38 @@ def view_tasks(request):
 
 
 def new_task(request):
-    if request.method == "POST":
-        form = TaskForm(request.user.id, request.POST)
-        taskForm = form.save(commit=False)
-        if form.is_valid():
-            try:
-                Task.create_task(taskForm.title, taskForm.created_by, taskForm.assignee, taskForm.priority,
-                                 taskForm.status, taskForm.description)
-
-                messages.success(request, 'Task was added successfully')
+    context = {'manager': False}
+    if not request.user.is_anonymous:
+        logged_user = User.objects.get(user=request.user)
+        context = {'user': logged_user}
+        if logged_user.is_manager():
+            color = 'red'
+            if request.method == "POST":
+                form = TaskForm(request.user.id, request.POST)
+                taskForm = form.save(commit=False)
+                if form.is_valid():
+                    try:
+                        Task.create_task(taskForm.title, taskForm.assignee, taskForm.created_by, taskForm.priority,
+                                         taskForm.status, taskForm.description)
+                        messages.success(request, 'Task was added successfully')
+                        color = 'green'
+                        form = TaskForm(request.user.id)
+                    except Exception as e:
+                        messages.warning(request, e)
+                        form = TaskForm(request.user.id,
+                                        initial={'title': taskForm.title, 'assignee': taskForm.assignee,
+                                                 'priority': taskForm.priority, 'status': taskForm.status,
+                                                 'description': taskForm.description})
```

Comment:
I think you can define a variable after line # 27 like:
initial_dict = {'title': taskForm.title, 'assignee': taskForm.assignee, 'priority': taskForm.priority, 'status': taskForm.status, 'description': taskForm.description}
instead of repeat the code of this dict twice (here and in lines # 43-46
and here to do it like:
```suggestion
                        form = TaskForm(request.user.id, initial=initial_dict)
```